### ISSUE TITLE
scons on osx fails with fatal error 'visual_server_wrap_mt.h' file not found

### ISSUE THREAD

**Issue by vichug** [godotengine/godot#7415]
fresh github godot, osx 10.10.5

After a 
`scons platform=osx`
it ends there : 
```
In file included from platform/osx/os_osx.mm:41:
platform/osx/os_osx.h:38:10: fatal error: 'servers/visual/visual_server_wrap_mt.h' file not found
#include "servers/visual/visual_server_wrap_mt.h"
         ^
1 error generated.
scons: *** [platform/osx/os_osx.osx.tools.32.o] Error 1
scons: building terminated because of errors.
```

**Comment by vichug** [comment]
i should probably add that my previous `git pull` dates back from around 26th of August

**Comment by vichug** [comment]
...after checking, there ain't a `servers/visual/visual_server_wrap_mt.h` file neither in the godot git repo nor on my computer. It seems like something forgotten ?

**Comment by BastiaanOlij** [comment]
it's there for me but its gone from the current stuff, most likely due to the gles3 work being merged, note the message Juan send around that the current main branch will be unstable for a little bit.

**Comment by akien-mga** [comment]
OSX is not yet configured to build the current master branch. There's a WIP PR that will need to be rebased and maybe further enhanced: #7177

**Comment by vichug** [comment]
yes, i saw the info about current master beeing unstable after posting this issue, so this may be noise, sorry for that !

**Comment by BastiaanOlij** [comment]
Visual server wrap is a class that wraps the visual server object for running in a separate thread. Because of the heavy changes in the visual server I guess Juan wasn't ready to make all the changes to the wrapper class and disabled it for the time being, but hadn't put the changes through to osx. working on it.

**Comment by BastiaanOlij** [comment]
@akien-mga this can be closed now with #7429 done and dusted?
